src/scene/graphics/scene-grab.js

```diff
@@ -123,19 +123,19 @@ class SceneGrab {
         return texture?.format ?? this.device.backBufferFormat;
     }
 
-    shouldReallocate(targetRT, sourceTexture, testFormat) {
+    shouldReallocate(targetRT, sourceRT, testFormat) {
 
         // need to reallocate if format does not match
         if (testFormat) {
             const targetFormat = targetRT?.colorBuffer.format;
-            const sourceFormat = this.getSourceColorFormat(sourceTexture);
+            const sourceFormat = this.getSourceColorFormat(sourceRT);
```

Comment:
I completely missed that this parameter is used to get the format from the texture .. so RT here won't work :(

Reply:
I guess so. Just make sure we're not using the format in the case you're changing, as we don't want to compare against color format.
split the line is ok .. just no lines with just a `(` on it like you had.